.docker/requirements_less.txt

```diff
@@ -0,0 +1,11 @@
+Flask_Caching>=1.7.2
```

Comment:
I'm leery of adding a second `requirements.txt` file as I'm not sure we'll remember to update it. What do you think about generating on the fly with something like awk to remove the gsi-qc-etl line?